assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -370,10 +381,13 @@ function getInnerElement(
     onClick,
     showVideoDetail,
     dropTargetsBindings,
+    newVideoRef,
```

Comment:
Could this be turned into an actual component rather than "just" a function? And maybe moved to a separate file?

Reply:
Changed in 28714f906386bfcdf7971350b5f344b91b637eb7, PTAL